common/oatbox/filesystem/utils/FlyWrapperTrait.php

```diff
@@ -5,12 +5,12 @@
 use League\Flysystem\Config;
 /**
  * A trait to facilitate creation of adapter wrappers
- * 
+ *
  * @author Joel Bout
  */
 trait FlyWrapperTrait
```

Comment:
`FlyWrapperTrait` seems a bit cryptic to me as a class name. Does it trap flies? ;-)
What do you think about `FlyFileSystemWrapperTrait`?